# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: toolkit/components/windowwatcher/nsWindowWatcher.cpp
# Commit: 2203b9e3146e
# Full Hash: 2203b9e3146ed5567b59718d8904ff77cfc352aa
# Author: Nika Layzell <nika@thelayzells.com>
# Date: 2020-04-08 03:36:50
# Regressor Bug: 1616353
# File Overlap Count: 1
# Description:
#   Bug 1616353 - Part 8: Remove forceInitialBrowserNonRemote, r=Gijs
#   
#   This method is no longer necessary, as frontend will always make the correct
#   decisions for the remoteness of the intial content browser by reading
#   openWindowInfo.
# ==============================================================================

diff -r fbc669bbf02f -r 2203b9e3146e toolkit/components/windowwatcher/nsWindowWatcher.cpp
--- a/toolkit/components/windowwatcher/nsWindowWatcher.cpp	Tue Apr 07 21:39:44 2020 +0000
+++ b/toolkit/components/windowwatcher/nsWindowWatcher.cpp	Tue Apr 07 21:39:47 2020 +0000
@@ -948,16 +948,6 @@
       }
 
       if (newChrome) {
-        nsCOMPtr<nsIAppWindow> appWin = do_GetInterface(newChrome);
-        if (appWin) {
-          nsCOMPtr<nsIXULBrowserWindow> xulBrowserWin;
-          appWin->GetXULBrowserWindow(getter_AddRefs(xulBrowserWin));
-          if (xulBrowserWin) {
-            nsPIDOMWindowOuter* openerWindow =
-                aForceNoOpener ? nullptr : parentWindow.get();
-            xulBrowserWin->ForceInitialBrowserNonRemote(openerWindow);
-          }
-        }
         /* It might be a chrome AppWindow, in which case it won't have
             an nsIDOMWindow (primary content shell). But in that case, it'll
             be able to hand over an nsIDocShellTreeItem directly. */